site/en/docs/extensions/mv3/override/index.md

```diff
@@ -69,30 +65,27 @@ Register an override page in the [extension manifest][3] like this:
 
 For `PAGE_TO_OVERRIDE`, substitute one of the following:
 
-- `bookmarks`
-- `history`
-- `newtab`
+- `"bookmarks"`
+- `"history"`
+- `"newtab"`
 
-## Tips {: #tips }
+## Best practices {: #tips }
 
-For an effective override page, follow these guidelines:
+* **Make your page quick and small.** <br/>Users expect built-in browser pages to open instantly. Avoid doing things that might take a long
+time, like synchronous fetches of network or database resources.
```

Comment:
```suggestion
time. Specifically, avoid accessing database resources synchronously. When making network resources, prefer [`fetch()`](https://developer.mozilla.org/docs/Web/API/fetch) over `XMLHttpRequest()`.
```

Reply:
I deliberately didn't link `XMLHttpRequest()`. While browsers aren't yet in a position to deprecate it, non one should be using it in new code.